packages/kbn-router-to-openapispec/src/generate_oas.test.util.ts

```diff
@@ -54,7 +54,7 @@ export const getRouterDefaults = () => ({
   handler: jest.fn(),
 });
 
-export const getVersionedRouterDefaults = () => ({
+export const getVersionedRouterDefaults = (bodySchema?: any) => ({
```

Comment:
@jeanfabrice What should be the type here?

Reply:
@maryam-saeidi Do you mean to mention @jloleysens instead?

Reply:
Yes, sorry for the wrong mention 🙈

Reply:
I added the types in a few commits